consensus/polybft/validator_metadata.go

```diff
@@ -124,6 +124,16 @@ func (as AccountSet) GetAddresses() []types.Address {
 	return res
 }
 
+// GetAddresses aggregates addresses as map for given AccountSet
+func (as AccountSet) GetAddressesAsMap() map[types.Address]struct{} {
```

Comment:
this is technically set not a map since value is empty struct and has only keys. Maybe better to name it 
GetAddressesAsSet?

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/955/commits/ff760fa6cd42947bf39e43e2011f5b56aff9d308